source/nodejs/adaptivecards/src/utils.ts

```diff
@@ -273,20 +273,17 @@ export function addCancelSelectActionEventHandler(element: HTMLElement) {
     };
 }
 
-export function SetDateOrTimePickerVisibility(input: HTMLInputElement, value: string | undefined, show: boolean) {
-    if (!!show) {
+export function updateDateOrTimePickerVisibility(input: HTMLInputElement, hover: boolean) {
```

Comment:
If you follow my recommendation to move this function to `card-elements.ts`, I would also suggest renaming it - you're not updating the visibility of the whole input as the current name suggests, but only of the buttons/icons that adorn the input. So maybe `updateInputAdornersVisibility`?